test/js/css-token-stream-value.js

```diff
@@ -0,0 +1,67 @@
+suite('CSSTokenStreamValue', function() {
+  test("CSSTokenStreamValue is a CSSTokenStreamValue and CSSStyleValue", function() {
+    assert.instanceOf(new CSSTokenStreamValue(), CSSTokenStreamValue);
+    assert.instanceOf(new CSSTokenStreamValue(), CSSStyleValue);
+  });
+
+  test('Constructor does not throw', function() {
```

Comment:
It would be better to assert that the constructed object contains the correct values, instead of just testing that it doesn't throw